airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/test/steps/common.py

```diff
@@ -513,7 +513,9 @@ def _get_command_options(self) -> List[str]:
             command_options += ["--should-read-with-state", self.should_read_with_state]
         if self.test_evaluation_mode:
             command_options += ["--test-evaluation-mode", self.test_evaluation_mode]
-        return command_options + ["--stream", self.selected_streams] if self.selected_streams else []
```

Comment:
This was a bug that was causing us to ignore command options when there were no selected streams.